src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
+  "name": "Sas Implementation Record",
+  "administratorId": "sas_admin/sas/0",
```

Comment:
SAS-SAS V1.0.0 and V1.5.1 has a format sas_admin/sas/$ADMINISTRATOR. Is this the recent change which is not yet reflected in the Spec?